daemon/daemon.go

```diff
@@ -462,18 +487,40 @@ func (d *Daemon) Start() {
 
 		return nil
 	})
+
+	// notify systemd that we are ready
+	systemdSdNotify("READY=1")
+}
+
+var execCommand = exec.Command
+
+var shutdownMsg = i18n.G("reboot scheduled to update the system")
+
+func reboot(rebootDelay string) error {
```

Comment:
This would be nicer as a time.Duration, stringified internally, so we don't need to handle strings and guess units/etc elsewhere.